docs/runtime/initializing-storage.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Initializing Runtime Storage"
+---
+
+Most runtime modules contain storage values that may be changed or updated as the blockchain runs and users interact with the module. The ways that data may change are coded in the runtime module's functions (both dispatchable and not), and are covered in several [tutorials](/tutorials).
+
+In some cases storage should be initialized with a value before any user has interacted with it. This is primarily the job of the genesis configuration. There are four ways runtime developers may specify initial values for their storage items, and this article will cover all four. Another article discusses how the genesis configuration works [behind the macros](runtime/types/genesisconfig-struct.md).
+
+## Hard-Code Specific Values
+
+The simplest way (and one that technically does not use genesis configuration) to initialize storage is simply to hard-code specific values for each parameter. This option is best when there are clear default values that all deployments of the module will want to use. The runtime developer can choose this option by including `config()` in the `decl_storage!` marco, and an initial value at the end of the line.
+
+Consider this example module where players ante to play a game and add to the pot by raising.
+
+```rust
+decl_storage! {
+  trait Store for Module<T: Trait> as TemplateModule {
+    // Points ante'd to play the game
+    pub Ante get(ante) config(): u32 = 5;
+
+    // Minimum raise on each turn
+    pub MinRaise get(min_raise) config(): u32 = 7;
+  }
+}
+```
+
+## Defer until Deploy Time with `config()`
+
+The second option available to runtime developers is to actually _not_ specify their own genesis values, but to leave that option up to whomever actually deploys the module in a blockchain. This option makes sense when different deployments of the module, may reasonably want different configuration values, and the various storage values do not need to maintain a particular relationship to one another. The runtime developer can choose this option by including `config()` in the `decl_storage!` marco, and nothing more.
+
+```rust
+decl_storage! {
+  trait Store for Module<T: Trait> as TemplateModule {
+    // Points ante'd to play the game
+    pub Ante get(ante) config(): u32;
+
+    // Minimum raise on each turn
+    pub MinRaise get(min_raise) config(): u32;
+  }
+}
+```
```

Comment:
yes exact, also note that `get(name)` is not mandatory here: if you don't specify a getter then you have to specify the name of the field of the genesis config struct inside config this way: `config(min_raise)`

Reply:
Oh! I didn't understand how that worked. I've clarified it in the callout a few lines lower.